src/apps/backend/modules/access-token/rest-api/account-auth-middleware.ts

```diff
@@ -1,46 +1,41 @@
-import { NextFunction, Request, Response } from 'express';
-import jsonwebtoken from 'jsonwebtoken';
+import _ from 'lodash';
 
-import ConfigService from '../../config/config-service';
 import {
-  AccessTokenExpiredError,
+  applicationController,
+  Request,
+  Response,
+  NextFunc,
+} from '../../application';
+import AccessTokenService from '../access-token-service';
+import {
   AuthorizationHeaderNotFound,
   InvalidAuthorizationHeader,
   UnAuthorizedAccessError,
 } from '../types';
 
-export default class AccountAuthMiddleware {
-  public static ensureAccess(
-    req: Request,
-    _res: Response,
-    next: NextFunction,
-  ): void {
+const AccountAuthMiddleware = {
```

Comment:
should this be `accountAuthMiddleware`, think it better aligns with JS naming convention.

Reply:
Addressed